sdk/signalr/arm-signalr/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@azure/arm-signalr",
   "author": "Microsoft Corporation",
   "description": "SignalRManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "dependencies": {
-    "@azure/ms-rest-azure-js": "^1.1.0",
-    "@azure/ms-rest-js": "^1.1.0",
+    "@azure/ms-rest-azure-js": "^1.3.2",
+    "@azure/ms-rest-js": "^1.8.1",
```

Comment:
We should be using version 2 of ms-rest-js and ms-rest-azure-js libraries.
If we re-generate the code using the right version of the generator like @amarzavery said, the above should get fixed

Reply:
```bash
autorest --reset
autorest --latest --typescript
autorest --info
AutoRest code generation utility [version: 2.0.4283; node: v10.15.3]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest


Showing All Installed Extensions

 Type       Extension Name                           Version      Location 
 core       @microsoft.azure/autorest-core           2.0.4390     /Users/amarz/.autorest/@microsoft.azure_autorest-core@2.0.4390
 extension  @microsoft.azure/autorest.modeler        2.3.51       /Users/amarz/.autorest/@microsoft.azure_autorest.modeler@2.3.51
 extension  @microsoft.azure/autorest.typescript     4.2.0        /Users/amarz/.autorest/@microsoft.azure_autorest.typescript@4.2.0


amarz-MBP:sdk amarz$ 
```

As you can see it installs 4.2.0 version of TS generator. After doing that, if you do **NOT** specify the `--use="@microsoft.azure/autorest.typescript@4.2.0` switch then it is fine. Since autorest will automatically use the currently  installed version of the TS generator

Reply:
@michaeljqzq - An update, please use 4.2.2 version of `@microsoft.azure/autorest.typescript` to generate the sdk.